single_tech_samples/databricks/sample3_cluster_provisioning_and_data_access/README.md

```diff
@@ -1 +1,117 @@
-# Placeholder
\ No newline at end of file
+# Provisioning Clusters and Access Data
+
+- [Provisioning Clusters and Access Data](#provisioning-clusters-and-access-data)
+  - [1. How to use this sample](#1-how-to-use-this-sample)
+    - [1.1 Prerequisites](#11-prerequisites)
+    - [1.2. Setup and deployment](#12-setup-and-deployment)
+    - [1.3. Cleanup](#13-cleanup)
+  - [2 Run sample with Sample 2 (enterprise Azure Databricks environment)](#2-run-sample-with-sample-2-enterprise-azure-databricks-environment)
+
+## 1. How to use this sample
+
+In this sample we are doing a few things:
+
+- Upload some sample data into a newly created container
+- Adding the logged in account as a Storage Blob Data Contributor to enable passthrough access
+- Deploy a default Databricks cluster and a High-Concurrency Databricks cluster
+- Accessing data using the default cluster by mounting the Data Lake using the Account Key
+- Accessing data using the high-concurrency cluster by mounting the Data Lake using AD Credential Passthrough
+
+### 1.1 Prerequisites
+
+1. You need to have already deployed the Databricks workspace using `sample1_basic_azure_databricks_environment`
+2. You need to have rights to assign account roles to the storage account
+
+### 1.2. Setup and deployment
+
+1. First you need to configure your environment variable file `.env`, which should look same
+    or similar to the one you are using in `sample1`. Then run:
+
+    ```bash
+    source .env
+    ```
+
+    This will give you the configuration you need to start the following process.
+
+    Then make sure you are logged in to your Azure account via:
+
+    ```bash
+    az login
+    ```
+
+    And follow the prompts to complete your login process.
+
+1. Run `deploy.sh`
+
+    You can run ./deploy.sh to start the deployment process in the below steps.
+
+    ```bash
+    ./deploy.sh
+    ```
+
+1. upload-sample-data.sh
```

Comment:
These should be children of Deploy.sh
upload-sample-data.sh
deploy-default-and-hc-clusters.sh
configure-databricks-cli.sh
import-data-access-notebooks.sh

Reply:
Yeah I think this was not very clear, which I've rectified by removing the running script.